packages/office-ui-fabric-react/src/components/MessageBar/examples/MessageBar.Styled.Example.tsx

```diff
@@ -0,0 +1,205 @@
+import * as React from 'react';
+import { MessageBarButton } from 'office-ui-fabric-react/lib/Button';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { Label } from 'office-ui-fabric-react/lib/Label';
+import { Stack, StackItem } from 'office-ui-fabric-react/lib/Stack';
+import { MessageBar, MessageBarType } from 'office-ui-fabric-react/lib/MessageBar';
+
+const log = (text: string): (() => void) => (): void => console.log(text);
+
+export const MessageBarStyledExample: React.StatelessComponent = () => (
+  <Stack gap={20}>
+    <StackItem>
+      <Label>Info/Default MessageBar - custom styles</Label>
+      <MessageBar
+        styles={{
```

Comment:
Inlining `styles` like this seems like something we shouldn't encourage in our examples. Maybe we could centralized them in a `const` object at the top using `mergeStyleSets`?

This also applies to the rest of the examples in this file.

Reply:
I guess I don't quite see the point of putting them in a const at the top if they're only used once?